pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/region_share.go

```diff
@@ -87,6 +104,62 @@ func (r *QoSRegionShare) TryUpdateProvision() {
 	}
 }
 
+// restrictProvisionControlKnob is to restrict provision control knob by reference policy
+func (r *QoSRegionShare) restrictProvisionControlKnob(originControlKnob map[types.CPUProvisionPolicyName]types.ControlKnob) map[types.CPUProvisionPolicyName]types.ControlKnob {
+	restrictedControlKnob := make(map[types.CPUProvisionPolicyName]types.ControlKnob)
+	for baseName, controlKnob := range originControlKnob {
```

Comment:
baseName and refName is confusing since they have similar meaning. baseName -> policyName(or some better name), refName -> refPolicyName might be better.